### ISSUE TITLE
Diff editor - aria label should advertise the use of F7 / shift + F7

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#158929]
Diff editor is not accessible unless the user uses the F7 / Shift + F7 experience.
We should make this more discoverable to the users. For example the aria label of a diff editor should at the end append "use F7 / shift + F7 to navigate changes"

fyi @hediet

**Comment by isidorn** [comment]
@meganrogge assigning to you and removing the milestone since this might be a good win and something that is within our reach.
Namely the diff editor is not that accessible unless users use the `F7` `shift + F7` experience to navigate between changes. So we should make this more discoverable by potentially mentioning it in the aria label of the diff editor.

**Comment by meganrogge** [comment]
Verify that when you are using a screen reader and enter a diff editor, you're informed about using shift + F7 navigation
